addons/survey/models/survey_question.py

```diff
@@ -609,9 +652,13 @@ def _get_stats_summary_data(self, user_input_lines):
         if self.question_type in ['simple_choice', 'multiple_choice']:
             stats.update(self._get_stats_summary_data_choice(user_input_lines))
         elif self.question_type == 'numerical_box':
-            stats.update(self._get_stats_summary_data_numerical(user_input_lines))
+            all_values = user_input_lines.filtered(lambda line: not line.skipped).mapped('value_numerical_box')
```

Comment:
Why moving the skipped check out of the method ? Other summary methods receive lines and then filters out irrelevant lines, isn't it ?